app/test/block_size_test.go

```diff
@@ -129,12 +129,13 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 			for _, hash := range hashes {
 				// TODO: once we are able to query txs that span more than two
 				// shares, we should switch to proving txs existence in the block
-				resp, err := queryWithOutProof(val.ClientCtx, hash)
+				resp, err := queryTx(val.ClientCtx, hash, true)
 				assert.NoError(err)
 				assert.Equal(abci.CodeTypeOK, resp.TxResult.Code)
 				if resp.TxResult.Code == abci.CodeTypeOK {
 					heights[resp.Height]++
 				}
+				require.True(resp.Proof.VerifyProof())
```

Comment:
Now we're requiring that all transactions queried for during the integration test have valid inclusion proofs.